SevenZip/SevenZipExtractor.cs

```diff
@@ -107,7 +107,7 @@ private void Init(Stream stream)
             SevenZipLibraryManager.LoadLibrary(this, _format);
             try
             {
-                _inStream = new ArchiveEmulationStreamProxy(stream, _offset);
+                _inStream = new ArchiveEmulationStreamProxy(stream, _offset, false);
```

Comment:
Maybe make the last parameter optional (`dispose = false`) to make the API backward compatible?